FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -32,7 +32,7 @@ @protocol FIRAppDistributionInstanceProvider <NSObject>
 @interface FIRAppDistribution () <FIRLibrary, FIRAppDistributionInstanceProvider>
 @property(nonatomic) BOOL isTesterSignedIn;
 
-@property(nullable, nonatomic) FIRAppDistributionUIService *UIService;
+@property(nullable, nonatomic) FIRAppDistributionUIService *uiService;
```

Comment:
Looking around, I think we should leave this as UIService

Reply:
Fixed!